### ISSUE TITLE
Beta does not seem to be able to import icons that have a hole in them

### ISSUE THREAD

**Issue by lbenedetto** [alexjlockwood/ShapeShifter#327]
It can't import this svg from the material icons library. I've also had trouble import other icons such as a search icon.

https://material.io/tools/icons/static/icons/baseline-location_on-24px.svg

Correctly imported by release version of ShapeShifter
![image](https://user-images.githubusercontent.com/4466272/52220382-fe6cd580-289e-11e9-914d-2e71c47a715b.png)

Failed import by ShapeShifter beta
![image](https://user-images.githubusercontent.com/4466272/52220415-18a6b380-289f-11e9-9848-c2ad144d7492.png)

The actual path strings after import are identical

**Comment by bhaarata** [comment]
Yes, I have the same – correctly imported by release version of ShapeShifter, and by beta was failed. Correct link to SVG:
https://material.io/resources/icons/?icon=location_on&style=baseline

Or here is the code from this SVG:
`<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0z" fill="none"/><path d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zm0 9.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5z"/></svg>`
